apps/console/src/features/server-configurations/pages/index.ts

```diff
@@ -18,4 +18,5 @@
 
 export * from "./admin-advisory-banner-page";
 export * from "./admin-session-advisory-banner-edit-page";
+export * from "./remote-logging";
```

Comment:
Shall we avoid exporting the remote-logging through the index file here? Any limitation here?